tests/test_background.py

```
@@ -0,0 +1,82 @@
+
+def setup_function(_):
+    _clean_bg()
+
+
+def teardown_function(_):
+    _clean_bg()
+
+
+def test_upload_and_get_background(live_server):
+    """上传 png → 200 + 返回 url；GET /api/background → 200 且为 PNG 字节。"""
+    resp = requests.post(live_server.url + '/api/upload_bg',
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)